WebDriverAgentLib/Commands/FBDebugCommands.m

```diff
@@ -40,7 +40,7 @@ + (NSArray *)routes
 
 + (id<FBResponsePayload>)handleGetSourceCommand:(FBRouteRequest *)request
 {
-  FBApplication *application = request.session.activeApplication ?: [FBApplication fb_activeApplication];
+  FBApplication *application = request.session.activeApplication;
```

Comment:
After this change, `[[FBRoute GET:@"/source"].withoutSession respondWithTarget:self action:@selector(handleGetSourceCommand:)],` start returning `Cannot get 'xml' source of the current application` error.
Do you have an idea to fix it?
(With session works fin 👍, btw)

Reply:
ah, I'll put it back. Thanks for the hint